packages/common/rollup-resolve-remap.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+const pathLib = require("path");
+
+/**
+ * Rollup plugin that remaps import module specifiers.
+ *
+ * Similar to @rollup/plugin-alias, but different in that it works better with
+ * relative path module specifiers, not just bare module specifiers.
+ *
+ * Instead of applying substitutions directly to the original module specifier
```

Comment:
Can you file a feature request against `@rollup/plugin-alias`?

Reply:
Will do.